docs/topics/proc_configuring-your-application-to-use-spring-boot.adoc

```diff
@@ -47,6 +47,29 @@ Include the following properties in your `pom.xml` file to track the version of
 ----
 --
 
+. Specify the repositories containing {ProductShortName} {SpringBoot} Starters and the {SpringBoot} Maven Plugin.
+
+[source,xml,subs="attributes+",options="nowrap"]
+----
+<!-- Specify the repositories containing RHOAR artifacts. -->
```

Comment:
I'm nitpicking, but I'd prefer this line be indented the same as the one below.

Reply:
@tradej Ah, of course, I'll fix this momentarily:)

Reply:
fixed:)